GetVisibleServices/handler.ts

```diff
@@ -41,27 +43,37 @@ export function GetVisibleServicesHandler(
 ): IGetVisibleServicesHandler {
   return async () => {
     const errorOrMaybeVisibleServicesJson = await getBlobAsObject(
-      t.dictionary(ServiceId, VisibleService),
+      t.record(t.string, VisibleService),
```

Comment:
Why not `ServiceId` instead of `t.string` ?

Reply:
This is actually very annoying: it doesn't work anymore using `ServiceId`.

Reply:
@balanza @michaeldisaro this is actually a good point to study. By leaving ServiceId , the resulting type was n empty decoder

Reply:
It's strange. AFAIK `t.record` accepts 2 Mixed codecs :(

Reply:
@gquadrati could this depend on the changes done to visible_service model? We defined the key as string.

Reply:
@michaeldisaro I don't think so, we use it after this step

Reply:
@AleDore I think something has changed 
![image](https://user-images.githubusercontent.com/75862507/131699538-ead0553c-08e1-491e-965b-f2b55b3cd2bd.png)

Reply:
Even if types are not correctly inferred, we can leave `ServiceId`. Done [here](https://github.com/pagopa/io-functions-app/pull/202/commits/2e1d3756d7c369229b9f4f878bc339128a3ec331)

Reply:
👍